src/main/java/io/appium/java_client/flutter/FlutterAndroidDriver.java

```diff
@@ -0,0 +1,134 @@
+    }
+
+    public FlutterAndroidDriver(URL remoteAddress, Capabilities capabilities) {
+        super(remoteAddress, capabilities);
+    }
+
+    public FlutterAndroidDriver(URL remoteAddress, HttpClient.Factory httpClientFactory, Capabilities capabilities) {
+        super(remoteAddress, httpClientFactory, capabilities);
+    }
+
+    public FlutterAndroidDriver(AppiumDriverLocalService service, Capabilities capabilities) {
+        super(service, capabilities);
+    }
+
+    public FlutterAndroidDriver(
+            AppiumDriverLocalService service, HttpClient.Factory httpClientFactory, Capabilities capabilities) {
+        super(service, httpClientFactory, capabilities);
+    }
+
+    public FlutterAndroidDriver(AppiumServiceBuilder builder, Capabilities capabilities) {
+        super(builder, capabilities);
+    }
+
+    public FlutterAndroidDriver(
+            AppiumServiceBuilder builder, HttpClient.Factory httpClientFactory, Capabilities capabilities) {
+        super(builder, httpClientFactory, capabilities);
+    }
+
+    public FlutterAndroidDriver(HttpClient.Factory httpClientFactory, Capabilities capabilities) {
+        super(httpClientFactory, capabilities);
+    }
+
+    public FlutterAndroidDriver(ClientConfig clientConfig, Capabilities capabilities) {
+        super(clientConfig, capabilities);
+    }
+
+    public FlutterAndroidDriver(AppiumClientConfig appiumClientConfig, Capabilities capabilities) {
+        super(appiumClientConfig, capabilities);
+    }
+
+    public FlutterAndroidDriver(Capabilities capabilities) {
+        super(capabilities);
+    }
+
+    public FlutterAndroidDriver(URL remoteSessionAddress, String automationName) {
+        super(remoteSessionAddress, automationName);
+    }
+
+    /**
+     * Waits for an element to become visible on the Flutter application.
+     *
+     * @param option WaitOptions specifying the element and optional timeout.
+     */
+    @Override
+    public void waitForVisible(WaitOptions option) {
+        CommonFlutterCommands.waitForVisible(this, option);
+    }
+
+    /**
+     * Waits for an element to become invisible on the Flutter application.
+     *
+     * @param option WaitOptions specifying the element and optional timeout.
+     */
+    @Override
+    public void waitForInVisible(WaitOptions option) {
+        CommonFlutterCommands.waitForInVisible(this, option);
+    }
+
+    /**
+     * Scrolls to make an element visible on the Flutter application.
+     *
+     * @param option ScrollOptions specifying the finder and scroll direction.
+     */
+    @Override
+    public void scrollTillVisible(ScrollOptions option) {
+        CommonFlutterCommands.scrollTillVisible(this, option);
+    }
+
+    /**
+     * Performs a double click on an element in the Flutter application.
+     *
+     * @param option DoubleClickOptions specifying the element to double-click.
+     */
+    @Override
+    public void performDoubleClick(DoubleClickOptions option) {
```

Comment:
Consider group commands to interfaces with default methods. later these interfaces could be inherited by both android and ios drivers to avoid unnecessary duplication